tests/runtest.sh

```diff
@@ -328,11 +336,29 @@ echo > ${__CoreRTTestBinDir}/testResults.tmp
 
 __BuildOsLowcase=$(echo "${CoreRT_BuildOS}" | tr '[:upper:]' '[:lower:]')
 __TestSearchPath=src/Simple/${CoreRT_TestName}
+
+__tizen_cross_prepare_for_tests() {
+        # Create copy libclrjit.so, this rename is hidden by the project
+        # make file in default building process.
+        # libclrjit.so is used by the CoreCLR that compiler runs on.
+        # libclrjitilc.so is used for the actual compilation.
+        cp ${__tizenToolsCoreFXRoot}/libclrjit.so ${__tizenToolsCoreFXRoot}/libclrjitilc.so
```

Comment:
The dependency on the live clrjit is also problematic for CI. It means that JIT-EE changes, or accidental break checked into CoreCLR, would cause CoreRT CI to fail.

Reply:
Yes, but what can we do?) in ./bin/Linux.armel.{build_type}/tools/ libclrjit.so is x64 version. Where and how can we get the current stable version?

Reply:
It is `linux-x64` because of the target RuntimeIndentifier is overwritten to `linux-x64` here: https://github.com/dotnet/corert/blob/master/src/ILCompiler/RyuJIT/RyuJIT.depproj#L13 . This is workaround for CoreRT not being on the "portable Linux" plan yet. It should not be hard to tweak this to preserve the architecture to get `linux-arm` or `tizen-arm` binary.

The Tizen binaries would need to be fetched from the Tizen NuGet feed (https://tizen.myget.org/gallery/dotnet-core) similar to how it is done in other places (e.g. https://github.com/dotnet/corefx/blob/master/cross/arm32_ci_script.sh#L237).